react/components/Alert/index.js

```diff
@@ -109,11 +137,17 @@ Alert.propTypes = {
   onClose: PropTypes.func,
   /** Time in ms to auto close the alert */
   autoClose: PropTypes.number,
+  /** Set focus to the first focusable element inside alert, which should be the "action" when available or the "close" button */
+  focusOnOpen: PropTypes.bool,
+  /** Define element to be focused when the alert is dimissed */
+  focusAfterClosed: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
```

Comment:
I think this prop is not needed, you can get `document.activeElement` and hold so when the user dismisses the alert its focuses on this element. Another thing to think about is when the user dismiss the alert, but the focus wasn't in the alert anymore, so you shouldn't mess with the current focused element

Reply:
Just talked to Klynger on discord, and he suggested the idea of replacing both these properties for a single property that receives an ENUM.

The enum would have three options:
- Focus on action or dismiss when the alert opens
- Do the above and also register the document.activeElement and return to it when dismiss is triggered

Reply:
After analyzing the behavior I noticed a few scenarios where changing the focus after closing may be problematic, so I've decided not to implement it.